synapse/storage/data_stores/main/schema/delta/59/00push_summary_unread_count.sql

```diff
@@ -0,0 +1,18 @@
+/* Copyright 2020 The Matrix.org Foundation C.I.C
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+-- Store the number of unread messages, i.e. messages that triggered either a notify
+-- action or a mark_unread one.
+ALTER TABLE event_push_summary ADD COLUMN unread_count BIGINT NOT NULL DEFAULT 0;
```

Comment:
I'm sort-of wondering if we need to prepopulate this. or make it nullable, with a value of null meaning "see notif_count". wdyt?

(as an aside: I dunno if you're aware, but adding a column with a default value required rebuilding the table before postgres 11. I think it's ok in this instance though.)

Reply:
> I dunno if you're aware, but adding a column with a default value required rebuilding the table before postgres 11

Oh, I wasn't aware of that.